Source/Public/Clear-BicepModuleCache.ps1

```diff
@@ -0,0 +1,115 @@
+function Clear-BicepModuleCache {
+    [CmdletBinding()]
+    param (
+        
+        [Parameter(ParameterSetName = 'Oci', Position = 1)]
+        [switch]$Oci,
+
+        [Parameter(ParameterSetName = 'Oci', Position = 2)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Registry,
+
+        [Parameter(ParameterSetName = 'Oci', Position = 3)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Repository,
+
+        [Parameter(ParameterSetName = 'TemplateSpecs', Position = 1)]
+        [switch]$TemplateSpecs,
+
+        [Parameter(ParameterSetName = 'TemplateSpecs', Position = 2)]
+        [ValidateNotNullOrEmpty()]
+        [string]$SubscriptionId,
+
+        [Parameter(ParameterSetName = 'TemplateSpecs', Position = 3)]
+        [ValidateNotNullOrEmpty()]
+        [string]$ResourceGroup,
+
+        [Parameter(ParameterSetName = 'TemplateSpecs', Position = 4)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Spec,
+
+        [Parameter(ParameterSetName = 'Oci', Position = 4)]
+        [Parameter(ParameterSetName = 'TemplateSpecs', Position = 5)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Version,
+
+        [Parameter(ParameterSetName = 'All', Position = 1)]
+        [switch]$All
+     
+    )
+    begin {
+        # Check if a newer version of the module is published
+        if (-not $Script:ModuleVersionChecked) {
+            TestModuleVersion
+        }       
+    }
+
+    process {
+        
+        if ($Oci) {            
+            $OciPath = Get-BicepNetCachePath -Oci 
+            $RepositoryPath = $Repository -replace '\\', '$'
+
+            if (($Registry) -and ($Repository) -and ($Version)) {
+                Remove-Item -Recurse -Path "$OciPath/$Registry/$RepositoryPath/$Version$"
```

Comment:
is the last `$` sign on purpose? `$Version$`
If it is, it should be escaped using a backtick to prevent it from ever being interpreted as a variable

Reply:
Yes, the path looks like this: `C:\Users\StefanIvemo\.bicep\br\azopsmodules.azurecr.io\hello$bosse$alias\v1$`

Reply:
Since this string is in double quotes I still think we should escape the $ sign to prevent any missinterpretations of it as a variable. Replace the string with
```PowerShell
Remove-Item -Recurse -Path "$OciPath/$Registry/$RepositoryPath/$Version`$"
```
And I'll be happy with it.